oak_functions_launcher/tests/integration_test.rs

```diff
@@ -159,185 +149,3 @@ async fn test_launcher_weather_lookup_virtual() {
-        .expect("client couldn't decrypt response");
-
-    assert_eq!(std::str::from_utf8(&response).unwrap(), "test_value");
-
-    launched_instance
-        .kill()
-        .await
-        .expect("Failed to stop launcher");
-}
-
-#[tokio::test]
-async fn test_load_large_lookup_data() {
-    let oak_functions_linux_fd_bin_path =
-        oak_functions_test_utils::build_rust_crate_linux("oak_functions_linux_fd_bin")
-            .expect("Failed to build oak_functions_linux_fd_bin");
-
-    let params = launcher::native::Params {
-        enclave_binary: oak_functions_linux_fd_bin_path.into(),
-    };
-
-    let max_chunk_size = ByteUnit::Kilobyte(2);
-
-    // Initialize with 1 chunk.
-    let entries_one_chunk = oak_functions_test_utils::create_test_lookup_data(max_chunk_size, 1);
-    let mut lookup_data_file = oak_functions_test_utils::write_to_temp_file(
-        &oak_functions_test_utils::serialize_entries(entries_one_chunk),
-    );
-    let lookup_data_config = LookupDataConfig {
-        lookup_data_path: lookup_data_file.path().to_path_buf(),
-        update_interval: None,
-        max_chunk_size,
-    };
-    let wasm_path = oak_functions_test_utils::build_rust_crate_wasm("key_value_lookup")
-        .expect("Failed to build Wasm module");
-    let status_one_chunk = oak_functions_launcher::create(
-        launcher::GuestMode::Native(params),
-        lookup_data_config,
-        wasm_path.into(),
-        1024,
-    )
-    .await;
-    assert!(status_one_chunk.is_ok());
-
-    let (launched_instance, connector_handle, _) = status_one_chunk.unwrap();
-    let mut client = OakFunctionsAsyncClient::new(connector_handle);
-
-    let lookup_data_config = LookupDataConfig {
-        lookup_data_path: lookup_data_file.path().to_path_buf(),
-        update_interval: None,
-        max_chunk_size,
-    };
-
-    // Write 2 chunks in lookup data.
-    let enteries_two_chunks = oak_functions_test_utils::create_test_lookup_data(max_chunk_size, 2);
-    let write_result = lookup_data_file.write_all(&oak_functions_test_utils::serialize_entries(
-        enteries_two_chunks,
-    ));
-    assert!(write_result.is_ok());
-    let status_two_chunks = update_lookup_data(&mut client, &lookup_data_config).await;
-    assert!(status_two_chunks.is_ok());
-
-    let enteries_four_chunks = oak_functions_test_utils::create_test_lookup_data(max_chunk_size, 4);
-    let write_result = lookup_data_file.write_all(&oak_functions_test_utils::serialize_entries(
-        enteries_four_chunks,
-    ));
-    assert!(write_result.is_ok());
-
-    // Write 4 chunks in lookup data.
-    let status_four_chunks = update_lookup_data(&mut client, &lookup_data_config).await;
-    assert!(status_four_chunks.is_ok());
-
-    launched_instance
-        .kill()
-        .await
-        .expect("Failed to stop launcher");
-}
-
-#[ignore = "too expensive"]
-#[tokio::test]
-async fn test_load_two_gib_lookup_data() {
-    let oak_functions_linux_fd_bin_path =
```

Comment:
this effectively removes the test for large amounts of lookup data. Lookup data functionality remains tested. Per discussion in https://github.com/project-oak/oak/issues/3738, this should be fine

Reply:
Is there no way we can keep the test---and the one above `test_load_large_lookup_data`? I am not sure I have all the context for the PR, but I am not sure we have tests for this elsewhere.

Reply:
I'll look into it!

Reply:
@mariaschett I updated the two tests to work with the KVM version and added them back to this PR. Could you have another look? :)